lib/license_finder/decisions.rb

```diff
@@ -185,7 +185,7 @@ def unname_project(txn = {})
 
     def inherit_from(filepath)
       decisions =
-        if filepath =~ %r{^https?://}
+        if filepath.is_a?(Hash)
```

Comment:
How about keep simply string based url support and add the new hash structure as an addition?

Reply:
Hi @forelabs , I did that on Friday but forgot to push.